library/src/main/java/com/michaldrabik/classicmaterialtimepicker/model/CmtpDate.kt

```diff
@@ -0,0 +1,22 @@
+package com.michaldrabik.classicmaterialtimepicker.model
+
+import java.util.*
+
+data class CmtpDate(
```

Comment:
Let's add init validation check - see CmtpTime12 and CmtpTime24 classes.

Reply:
Ok. I'll check for month and day. But it doesn't make much sense to check for years too.